geps/gep-1767/metadata.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: internal.gateway.networking.k8s.io/v1alpha1
 kind: GEPDetails
 number: 1767
 name: CORS Filter
-status: Provisional
+status: Experimental
```

Comment:
Thanks @GunaKKIBM! Do you mind making a corresponding change in mkdocs.yml as well?

Reply:
Done. Thank you!